### DISCUSSION TITLE
Job runs twice in Azure AKS node

### DISCUSSION THREAD

**Discussion by nirab25** [quartznet/quartznet#2000] [Q&A]
**Describe the bug**
I need to run a job daily at 11:00 AM UTC.
The issue is Job runs twice instead of once 

Job run once in local machine but when deployed to AKS Cluster node, it runs twice at same time.
And for first iteration it runs 1, from 2nd day it runs twice

**Version used**
Quartz.AspNetCore Version="3.6.2"

With which version did you encounter this behavior.
Quartz.AspNetCore Version="3.5.0"

**To Reproduce**
Job Schedules using cron expression to run daily once but it runs twice. 
Please see the code below
Ideally code samples or event unit test.
public static void ConfigureJobScheduling(this IServiceCollection services, JobSchedulingSettings jobSchedulingSettings)
        {
            services.Configure<QuartzOptions>(options =>
            {
                options.Scheduling.IgnoreDuplicates = true;
                options.Scheduling.OverWriteExistingData = true;
                options.Scheduling.ScheduleTriggerRelativeToReplacedTrigger = true;
            });

            services.AddQuartz(q =>
            {
                q.UseMicrosoftDependencyInjectionJobFactory();
                q.UseSimpleTypeLoader();
                q.UseInMemoryStore();
                q.UseDefaultThreadPool(tp =>
                {
                    tp.MaxConcurrency = 10;
                });

                q.SetProperty("quartz.jobStore.clustered", "true");
                q.SetProperty("quartz.jobStore.acquireTriggersWithinLock", "true");

                // http://www.cronmaker.com/?1
                q.ScheduleJob<DailyEmailingExamReportJobEnabled>(trigger => trigger
                        ..WithCronSchedule("0 0 11 1/1 * ? *")); // run every 11:00 AM UTC
            });

            services.AddQuartzHostedService(options =>
            {
                // when shutting down we want jobs to complete gracefully
                options.WaitForJobsToComplete = true;
                options.AwaitApplicationStarted = true;
            });
        }
}
**Expected behavior**
Job should run only once at a time (at expected schedule)

Needs to run once daily at 11AM
**Additional context**

**Comment by nirab25** [comment]
Hi.. any one resolved this issue?
TIA

**Comment by dm-CaT** [comment]
Maybe in the cluster, you have two instances of service hosting the scheduler? In this case, you have to use some DB store, not an In-memory store.
